NBA-Stats-Dashboard/src/components/PlayerStats.jsx

```diff
@@ -1,10 +1,90 @@
-function PlayerStats() {
-    return (
-      <div>
-        <h1>PlayerStats</h1>
+import { useState, useEffect } from "react";
+import axios from "axios";
+import {
+  TopScorersUrl,
+  TopAssistsUrl,
+  TopReboundsUrl,
+} from "../utils/apiEndpoints";
+import "chart.js/auto";
+import { Chart, Tooltip, Legend } from "chart.js";
+import { Bar } from "react-chartjs-2";
+import Dropdown from "./Dropdown";
+
+Chart.register(Tooltip, Legend);
+const options = [
+  {
+    title: "Total Points",
+    url: TopScorersUrl,
+  },
+  {
+    title: "Total Assists",
+    url: TopAssistsUrl,
+  },
+  {
+    title: "Total Rebounds",
+    url: TopReboundsUrl,
+  },
+];
+
+const PlayerStats = () => {
+  const [leagueLeaders, setLeagueLeaders] = useState([]);
+  const [criteria, setCriteria] = useState(options[0]);
```

Comment:
Consider using this criteria state variable instead of the hardcoded "Goals" in the label property of the chart.